nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/utils/SNMPUtils.java

```diff
@@ -117,45 +122,48 @@ public static FlowFile updateFlowFileAttributesWithPduProperties(PDU response, F
                 }
                 flowFile = processSession.putAllAttributes(flowFile, attributes);
             } catch (Exception e) {
-                logger.warn("Failed to update FlowFile with AMQP attributes", e);
+                LOGGER.warn("Failed to update FlowFile with AMQP attributes", e);
```

Comment:
Although this was an existing message, should it reference SNMP instead of AMQP?
```suggestion
                LOGGER.warn("Failed to update {} with SNMP attributes", flowFile, e);
```